Functions/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# v8.7.1
```

Comment:
Should either be part of 8.7.0 or 8.6.1 if you want to cherry-pick to a `release-8.6` branch created from the 8.6.0 tag to patch the current release.